test/test-mda-models/aissemble-test-data-delivery-pyspark-model/tests/features/environment.py

```diff
@@ -75,7 +68,7 @@ def before_feature(context, feature):
             f"Starting services defined in Docker Compose file at {test_staging_path}"
         )
 
-        compose = DockerCompose(test_staging_path, select_docker_compose_arch())
+        compose = DockerCompose(test_staging_path, "docker-compose.yml")
```

Comment:
I: We don't need different docker compose files anymore since we don't need to specify the system architecture for the docker image